_posts/2019-07-09-10-praw-uzytecznego-designu.md

```diff
@@ -0,0 +1,124 @@
+
+<div class="img-with-legend">
+<img alt="Dashboard ze statystykami wyświetlający się po zalogowaniu do platformy Udemy" src="/assets/img/posts/2019-07-09-design/4.jpg" />
+<span class="img-legend">Rys. 5. Strona główna serwisu Udemy<br />źródło: <a href="https://www.udemy.com/">udemy.com</a> - dostęp: 2019-07-07</span>
+</div>
+
+## 5. Efekt pierwszeństwa i świeżości
+
+To tak naprawdę dwie zasady, mówiące o tym, że najlepiej zapamiętujemy to, co znajduje się na początku (efekt pierwszeństwa) oraz na końcu (efekt świeżości). Taki zabieg stosuje Allegro, które przedstawia oferty sponsorowane na szczycie i u dołu listy.
+
+<div class="img-with-legend">
+<img alt="Portal Allegro - Oferty sponsorowane na szczycie listy produktów" src="/assets/img/posts/2019-07-09-design/5a.jpg" />
+<span class="img-legend">Rys. 6. Portal Allegro - Oferty sponsorowane na szczycie listy produktów<br />źródło: <a href="https://allegro.pl/kategoria/meble-ogrodowe-fotele-wiszace-257757">Fotele wiszące na Allegro</a> - dostęp: 2019-07-07</span>
+</div>
+
+<div class="img-with-legend">
+<img alt="Portal Allegro - Oferty sponsorowane na dole listy produktów" src="/assets/img/posts/2019-07-09-design/5b.jpg" />
+<span class="img-legend">Rys. 7. Portal Allegro - Oferty sponsorowane na dole listy produktów<br />źródło: <a href="https://allegro.pl/kategoria/meble-ogrodowe-fotele-wiszace-257757">Fotele wiszące na Allegro</a> - dostęp: 2019-07-07</span>
+</div>
+
+## 6. Prawo Millera
+
+Według badań opublikowanych w 1956 roku w artykule "Magiczna liczba siedem plus minus dwa..." przez Georga A. Millera człowiek jest w stanie zapamiętać 7 informacji naraz. Dobrym przykładem wykorzystania tego prawa jest serwis Netflix. Po wejściu na stronę główną serwisu zostaniemy zasypani propozycjami filmówi i seriali, które z pewnością umilą nam niedzielny wieczór. Projektanci serwisu, zadbali o to, abyśmy nie zagineli w gąszczu propozycji, zostały one podzielone na kategorie, a każda z nich z kolei kusi 6 pozycjami.  
+
+<div class="img-with-legend">
+<img alt="Strona główna serwisu Netflix" src="/assets/img/posts/2019-07-09-design/6.jpg" />
+<span class="img-legend">Rys. 8. Strona główna serwisu Netflix<br />źródło: <a href="https://www.netflix.com/">netflix.com</a> - dostęp: 2019-07-07</span>
+</div>
+
+## 7. Prawo Teslera
+
+Stwierdza, że dla każdego systemu istnieje pewna złożoność, której nie można zmniejszyć. Zbyt duża liczba koniecznych działań może nas skutecznie zniechęcić do skorzystania z niektórych usług, jak chociażby wypełnianie mnóstwa pól formularza w sklepie internetowym podczas zakupu wymarzonej pary butów. Niestety w przypadku, gdy chcemy zarezerwować idealne miejsce na weekendowy pobyt, bez podania tych wszystkich informacji jak np. dokąd chcemy lecieć, na jak długo czy kiedy planujemy wrócić, system nie domyśli się, że chcielibyśmy spędzić tydzień w igloo na mroźnej północy Norwegii.
```

Comment:
Zamieniłbym kolejność drugiego zdania na " Zbyt duża liczba koniecznych działań, **jak chociażby wypełnianie mnóstwa pól formularza w sklepie internetowym podczas zakupu wymarzonej pary butów**, może nas skutecznie zniechęcić do skorzystania z niektórych usług."

Powinien być przecinek po "Niestety". Nie powinno być przecinka po słowie "pobyt".
Fragment "jak np. dokąd chcemy lecieć [...]" powinien być traktowany jako wtrącenie? Jeśli tak, to powinniśmy go otoczyć przecinkami (czyli dodać jeden przed słówkiem "jak", bo po "wrócić" już jest). Brakuje też przecinka wewnątrz tego wtrącenia, po słowie "długo".